text/2978-stack_vec.md

```diff
@@ -0,0 +1,185 @@
+- Feature Name: `stack_vec`
+- Start Date: 2020-09-27
+- RFC PR: [rust-lang/rfcs#0000](https://github.com/rust-lang/rfcs/pull/0000)
+- Rust Issue: [rust-lang/rust#0000](https://github.com/rust-lang/rust/issues/0000)
+
+# Summary
+[summary]: #summary
+
+This RFC proposes the creation of a new "growable" vector named `StackVec` that manages stack memory and can be seen as an alternative for the built-in structure that handles heap-allocated memory, aka `alloc::vec::Vec<T>`.
+
+# Motivation
+[motivation]: #motivation
+
+`core::collections::StackVec<T>` has several use-cases and should be conveniently added into the standard library due to its importance.
```

Comment:
`core::collections` does not currently exist, so that should probably be called out explicitly (it makes sense to put this structure there, but it would be a first)